hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azure/AzureBlobStorageTestAccount.java

```diff
@@ -21,6 +21,7 @@
 import com.microsoft.azure.storage.*;
 import com.microsoft.azure.storage.blob.*;
```

Comment:
test cases to assert rename and delete pass with redirection enabled needed.

Reply:
These tests are needed only for nonHNS + 0Auth , combination, test can check and skip run if these conditions are not qualified.

Reply:
Added test cases but not specified to run only for this specific combination, as want to validate that redirection works with all auth types